### PR TITLE
Resolve no umbrella header warning and standardize documentation

### PR DESCRIPTION
**Pull request by scotteg**

### CODE REVIEW

**Comment by ashfurrow** [comment]
Looks great! @thanegill?

**Comment by thanegill** [comment]
Looks great! Can your remove your trailing whitespace first @scotteg?

File: Example/Examples.playground/Contents.swift [hunk 1]
```diff
@@ -3,28 +3,28 @@ import RxCocoa
...
 
-//: All operators are available on Driver as well unless otherwise marked.
+//: All operators are also available on `Driver`, unless otherwise noted.
 
 
 //: ## Optionals
 
 example("filterNil") {
-    let _ = Observable<String?>
+    _ = Observable<String?>
```
Comment (thanegill) [inline Example/Examples.playground/Contents.swift]: Didn't know you could omit `let` in this case, very cool! 👍

Comment (scotteg) [inline Example/Examples.playground/Contents.swift]: @thanegill Sure, glad to. Do you just mean the indented spacing on blank lines? Such as...
screenshot 2016-03-26 11 49 11

Comment (thanegill) [inline Example/Examples.playground/Contents.swift]: Yes!

Comment (scotteg) [inline Example/Examples.playground/Contents.swift]: Done! I also updated README to match edits I made in the playground. And thanks for creating these extensions. I was just discussing what a pita it is to work with optionals in Rx the other day and these will really help out.

File: Example/Examples.playground/contents.xcplayground [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<playground version='5.0' target-platform='ios' auto-termination-delay='1' display-mode='rendered'>
+<playground version='5.0' target-platform='ios' auto-termination-delay='1' display-mode='raw'>
```
Comment (thanegill) [inline Example/Examples.playground/contents.xcplayground]: Can you change this back?

Comment (scotteg) [inline Example/Examples.playground/contents.xcplayground]: Oops! Done.

**Comment by thanegill** [comment]
Squash back to two commits and I'll merge!

**Comment by thanegill** [comment]
Thanks @scotteg!

**Pull request merged**